content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,301 @@
+---
+title: Group Export | R4 API
+---
+
+# Group Export
+
+<%= beta_tag %>
+
+* TOC
+{:toc}
+
+## Overview
+
+The Group Export operation is used to obtain FHIR resources for all members of a specified group of patients.
+
+An Export process uses the [FHIR Asynchronous Request Pattern](https://hl7.org/fhir/R4/async.html) to kick off the export job for a group of patients. Once the export job is kicked off, the status endpoint may be called to check the progress of the job. Once the job is complete, the status endpoint returns a list of files which may be downloaded using the file retrieval endpoint. After the appropriate ndjson files are downloaded, the delete endpoint should be called to clean up the no longer needed files.
+
+By default, the Group Export operation exports resources required for [United States Core Data for Interoperability (USCDI)](https://www.healthit.gov/isa/united-states-core-data-interoperability-uscdi). For a full list of resources see the [US Core Profiles](https://hl7.org/fhir/us/core/#us-core-profiles). When the _type parameter is used it returns only the FHIR resources specified. The _type parameter should be used to filter the resources required for a specific use case. Limiting the number of requested resources in the _type parameter is recommended as it can decrease response times.
+
+Client application developers will need to work with healthcare organizations to be granted access to the Group $export operation and to be provided with Groups to be exported. Organizations can manage Groups within [Ignite Management Tooling](https://cernercentral.com/ignite-management).
```

Comment:
So when I click this link for ignite management tooling it doesn't resolve probably because it's not enabled yet. It made me think that 3rd party developers won't have access to Ignite Management Tooling or reference pages so do we need a link here?

Reply:
I removed the link [here](https://github.com/cerner/fhir.cerner.com/pull/899/commits/ee37ee8e4678c8be7160f7ce7d603ef63930b8ac#diff-6963b85421ccad81bf2c26b848074843817dfa021595650e04f7ff89e1e85e31R24)